Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -32,41 +31,27 @@
 /**
  * The default web router
  *
- * @Flow\Scope("singleton")
  * @api
  */
 class Router implements RouterInterface
 {
     /**
-     * @Flow\Inject(name="Neos.Flow:SystemLogger")
```

Comment:
There is an `injectLogger` method already. I don't even know which wins currently.
But you're right, this is not directly related and should be put in a separate PR

Reply:
Yes, see the discussion thread in the PR above - I finally kept most of the `injectLogger()` methods to avoid having to rewrite all those tests that depended on that method.